python/orca/src/bigdl/orca/learn/pytorch/training_operator.py

```diff
@@ -451,12 +453,14 @@ def models(self):
     @property
     def optimizer(self):
         """First or only optimizer(s) created by the ``optimizer_creator``."""
-        return self._optimizers[0]
+        if self._optimizers:
+            return self._optimizers[0]
 
     @property
     def optimizers(self):
         """List of optimizers created by the ``optimizer_creator``."""
-        return self._optimizers
+        if self._optimizers:
+            return self._optimizers
```

Comment:
return None if no optimizer?

Reply:
Since we have assumed optimizers must be in train, we could remove checking the existence.